kalliope/core/Models/settings/Settings.py

```diff
@@ -38,6 +42,7 @@ def __init__(self,
         self.resources = resources
         self.variables = variables
         self.machine = platform.machine()   # can be x86_64 or armv7l
+        logger.debug("Current processor type: %s" % self.machine)
```

Comment:
Not a big deal, but since we do not support python 2, we should stop using the `%` syntax to format strings but `.format` instead...

Reply:
Exactly. This will be a part of the refactoring when pushing out python 2